docs/_includes/tests/textbox.html

```diff
@@ -117,3 +117,13 @@ <h2>Icon End, RTL</h2>
         </span>
     </div>
 </div>
+
+<h2>Actionable Icon</h2>
+<span class="textbox textbox--icon-end">
+    <input class="textbox__control" type="text" placeholder="placeholder text" />
+    <button class="icon-btn" type="button" aria-label="Choose Contact">
+        <svg aria-hidden="true" class="textbox__icon" focusable="false" width="16" height="16">
```

Comment:
@ianmcburnie Should we use the icon width/height here?
Like for messages: `<svg class="icon icon--messages">`

Reply:
I was thinking the same thing. It would be ideal in terms of BEM. But then without a fixed width I'm not sure we can know how much space to reserve at the end of the textbox...

Reply:
Well all icons max width/height are 24px so we know that. If we use the icon width/height classes, then it will just center the icon. 
We might want to change it for small icons which are 16x16. But maybe in future.

Reply:
That sounds good Andrew. @raveenakothapally Can you please look into seeing if that works?

Reply:
Sure, will check